great_expectations/experimental/datasources/spark_datasource.py

```diff
@@ -99,7 +99,7 @@ def test_connection(self, test_assets: bool = True) -> None:
         """
         if not self.base_directory.exists():
             raise TestConnectionError(
-                f"Path: {self.base_directory.resolve()} does not exist."
+                f"base_directory path: {self.base_directory.resolve()} does not exist."
```

Comment:
@NathanFarmer This test can now be done through the `DataConnector` at the file-path level.